server/http_handler.go

```diff
@@ -1083,11 +1083,9 @@ func (h schemaHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		// all table schemas in a specified database
 		if schema.SchemaExists(cDBName) {
 			tbs := schema.SchemaTables(cDBName)
-			tbsInfo := make([]*model.TableInfo, len(tbs))
-			for i := range tbsInfo {
-				tbsInfo[i] = tbs[i].Meta()
+			for _, tb := range tbs {
+				writeData(w, tb.Meta())
```

Comment:
should we write `[]*model.TableInfo{tb.Meta()}` to keep compatibility ?

Reply:
Would it break compatibility? I tried KeyVisualizer and it worked.

Reply:
Good catch, I will fix it